services/core-api/app/api/now_applications/resources/now_application_import_resource.py

```diff
@@ -61,6 +67,26 @@ def post(self, application_guid):
         application.longitude = longitude
         application.now_application_guid = application_guid
 
+        if contacts is not None:
+            for contact in contacts:
+
+                now_party = Party.find_by_party_guid(contact['party_guid'])
+                if not now_party:
+                    raise NotFound('No party found for party with guid ' + contact['party_guid'])
+
+                mine_party_appt_type = MinePartyAppointmentType.find_by_mine_party_appt_type_code(
+                    contact['mine_party_appt_type_code'])
+                if not mine_party_appt_type:
+                    raise NotFound('No mine party appointment type found for type with code ' +
```

Comment:
I think we prefer the `f''` syntax for concatenating strings but I'm not gonna nitpick that hard!